### ISSUE TITLE
KeyError: 'Tags'

### ISSUE THREAD

**Issue by RolandOtta** [binxio/ec2-volume-manager#2]
we have the ec2 volume manager up and running in one of our aws accounts.

even it seems to work fine we have from time to time the following error in the logs and i am not sure about any possible impacts

```
[ERROR] KeyError: 'Tags'
Traceback (most recent call last):
  File "/var/task/ec2_volume_manager/manager.py", line 211, in handler
    if not instance.attachment_name:
  File "/var/task/ec2_volume_manager/ec2_instance.py", line 25, in attachment_name
    return self.tags.get("ec2-volume-manager-attachment", None)
  File "/var/task/ec2_volume_manager/ec2_instance.py", line 33, in tags
    return {t["Key"]: t["Value"] for t in self["Tags"]}
```

please advise how to fix this error.

**Comment by mvanholsteijn** [comment]
Hi @RolandOtta, good to hear you are using the volume manager! I fixed the error in release 0.1.3.

**Comment by RolandOtta** [comment]
Hi @mvanholsteijn 

thank you you for the quick fix. verified it in our environment. looks good now!
